Week06/lesson12/homework/rock_paper_scissors_starter/js/main.js

```diff
@@ -0,0 +1,64 @@
+var botOptions = ["rock", "paper", "scissors"];
+
+var humanScore = 0;
+
+var computerScore = 0;
+
+function humanRock (e) {
+  var humanChoice = $("#rock").attr("id");
```

Comment:
While it's very cool that you pulled out the ID like this here, you might as well use a string `"rock"`. And if that's the case, then `humanChoice` is simple enough to just be passed directly into the function without need for a variable: `compare("rock", botChoice)`